### PR TITLE
Set some flag values true in local scripts

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Set some flag values true in local scripts

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/734

### CODE REVIEW

File: start_servers_afan_local.sh [hunk 1]
```diff
@@ -5,17 +5,54 @@ printf "\n[[[[[ start_servers_afan_local.sh ]]]]]\n\n"
 # PARENT CHAIN
 CONSOLE_LOG=true BLOCKCHAIN_DATA_DIR=~/ain_blockchain_data node ./tracker-server/index.js &
 sleep 5
-ACCOUNT_INDEX=0 P2P_PEER_CANDIDATE_URL='' CONSOLE_LOG=true ENABLE_DEV_CLIENT_SET_API=true ENABLE_TX_SIG_VERIF_WORKAROUND=true ENABLE_GAS_FEE_WORKAROUND=true BLOCKCHAIN_DATA_DIR=~/ain_blockchain_data node ./client/index.js &
+ACCOUNT_INDEX=0 \
```
Comment (liayoo) [inline start_servers_afan_local.sh:8]: ahhhh this is so nice!

File: start_servers_afan_local.sh [hunk 2]
```diff
@@ -29,7 +66,18 @@ do
     sleep 20
 done
 
-BLOCKCHAIN_CONFIGS_DIR=blockchain-configs/afan-shard PORT=9002 P2P_PORT=6002 ACCOUNT_INDEX=1 CONSOLE_LOG=true ENABLE_DEV_CLIENT_SET_API=true ENABLE_TX_SIG_VERIF_WORKAROUND=true ENABLE_GAS_FEE_WORKAROUND=true BLOCKCHAIN_DATA_DIR=~/ain_blockchain_data node ./client/index.js &
+BLOCKCHAIN_CONFIGS_DIR=blockchain-configs/afan-shard PORT=9002 P2P_PORT=6002 ACCOUNT_INDEX=1 CONSOLE_LOG=true ENABLE_DEV_CLIENT_SET_API=true ENABLE_JSON_RPC_TX_API=true ENABLE_REST_FUNCTION_CALL=true ENABLE_TX_SIG_VERIF_WORKAROUND=true ENABLE_GAS_FEE_WORKAROUND=true BLOCKCHAIN_DATA_DIR=~/ain_blockchain_data node ./client/index.js &
```
Comment (liayoo) [inline start_servers_afan_local.sh:69]: could you use multi lines for this command as well?

File: start_servers_local.sh [hunk 3]
```diff
@@ -29,15 +70,39 @@ do
     sleep 20
 done
 
-BLOCKCHAIN_CONFIGS_DIR=blockchain-configs/afan-shard PORT=9002 P2P_PORT=6002 ACCOUNT_INDEX=1 STAKE=100000 CONSOLE_LOG=true ENABLE_DEV_CLIENT_SET_API=true ENABLE_TX_SIG_VERIF_WORKAROUND=true ENABLE_GAS_FEE_WORKAROUND=true BLOCKCHAIN_DATA_DIR=~/ain_blockchain_data node ./client/index.js &
+BLOCKCHAIN_CONFIGS_DIR=blockchain-configs/afan-shard PORT=9002 P2P_PORT=6002 ACCOUNT_INDEX=1 STAKE=100000 CONSOLE_LOG=true ENABLE_DEV_CLIENT_SET_API=true ENABLE_JSON_RPC_TX_API=true ENABLE_REST_FUNCTION_CALL=true ENABLE_TX_SIG_VERIF_WORKAROUND=true ENABLE_GAS_FEE_WORKAROUND=true BLOCKCHAIN_DATA_DIR=~/ain_blockchain_data node ./client/index.js &
```
Comment (liayoo) [inline start_servers_local.sh:73]: ditto

File: start_servers_local.sh [hunk 4]
```diff
@@ -51,7 +116,19 @@ do
     sleep 20
 done
 
-BLOCKCHAIN_CONFIGS_DIR=blockchain-configs/sim-shard PORT=9012 P2P_PORT=6012 ACCOUNT_INDEX=1 STAKE=100000 CONSOLE_LOG=true ENABLE_DEV_CLIENT_SET_API=true ENABLE_TX_SIG_VERIF_WORKAROUND=true ENABLE_GAS_FEE_WORKAROUND=true BLOCKCHAIN_DATA_DIR=~/ain_blockchain_data node ./client/index.js &
+BLOCKCHAIN_CONFIGS_DIR=blockchain-configs/sim-shard PORT=9012 P2P_PORT=6012 ACCOUNT_INDEX=1 STAKE=100000 CONSOLE_LOG=true ENABLE_DEV_CLIENT_SET_API=true ENABLE_JSON_RPC_TX_API=true ENABLE_REST_FUNCTION_CALL=true ENABLE_TX_SIG_VERIF_WORKAROUND=true ENABLE_GAS_FEE_WORKAROUND=true BLOCKCHAIN_DATA_DIR=~/ain_blockchain_data node ./client/index.js &
```
Comment (liayoo) [inline start_servers_local.sh:119]: ditto

**Review by liayoo — APPROVED** [review]
LGTM with some comments. Thanks!

File: start_servers_afan_local.sh (hunk 2 above)
Comment (platfowner) [reply start_servers_afan_local.sh]: oh, I just missed it. Just fixed, thanks!

File: start_servers_local.sh (hunk 3 above)
Comment (platfowner) [reply start_servers_local.sh]: done, thanks!

File: start_servers_local.sh (hunk 4 above)
Comment (platfowner) [reply start_servers_local.sh]: done, thanks!

**Pull request merged**